src/synth/fluid_synth.c

```diff
@@ -4629,6 +4628,9 @@ fluid_synth_check_finished_voices(fluid_synth_t *synth)
             else if(synth->voice[j]->overflow_rvoice == fv)
             {
                 fluid_voice_overflow_rvoice_finished(synth->voice[j]);
+
+                /* Decrement voice count */
+                synth->active_voice_count--;
```

Comment:
For consistency I think it would be better to move this call into `fluid_voice_overflow_rvoice_finished()`.

Reply:
But I see you did it earlier and the unit tests failed. Perhaps you could just add a small comment to explain why the decrement must be put here.